### PR TITLE
Fix 'Your app version is missing important stock trading updates.'  + 24 hours stock market trading

### PR DESCRIPTION
**Pull request by henryzhangpku**

### CODE REVIEW

File: robin_stocks/robinhood/helper.py [hunk 1]
```diff
@@ -349,6 +349,8 @@ def request_post(url, payload=None, timeout=16, json=False, jsonify_data=True):
                 'Content-Type', 'application/x-www-form-urlencoded; charset=utf-8')
         else:
             res = SESSION.post(url, data=payload, timeout=timeout)
+        if res.status_code in [500,400]:
```
Comment (sanathnair09) [inline robin_stocks/robinhood/helper.py:352]: More of a code styling thing but I would do [400, 500] as it logically makes sense

File: robin_stocks/robinhood/helper.py [hunk 2]
```diff
@@ -349,6 +349,8 @@ def request_post(url, payload=None, timeout=16, json=False, jsonify_data=True):
...
+            raise Exception("Received "+ str(res.status_code))
```
Comment (sanathnair09) [inline robin_stocks/robinhood/helper.py:353]: Might also want to print the [res.reason](https://requests.readthedocs.io/en/latest/api/#requests.Response.reason) for additional debugging information

**Review by sanathnair09 — COMMENTED** [review]
Code wise everything seems good. Couldn't get to testing the actual changes since markets are closed might take a look later.

**Comment by henryzhangpku** [comment]
> Code wise everything seems good. Couldn't get to testing the actual changes since markets are closed might take a look later.

Thanks for the review. You can actually test the functionality in 24 hours market(new from RHS) instead of waiting for regular opens. It is offered to 43 stocks/etf now, and only allowed limit orders …

**Comment by eliwelch17** [comment]
You have all this market hours=stuff to make the extended hours trade amount round to a whole integer, you don't need it. I know it says on the RH website that after hours trading is for whole shares but you can still trade with fractional shares.

**Review by nkchittu — APPROVED** [review]

**Comment by paraluke23** [comment]
Hi, 3.0.5 doesn't fix the issue: Error in request_post: Expecting value: line 1 column 1 (char 0)

**Comment by Mariani-code** [comment]
Also Experiencing the same issues on 3.0.5 still with market sell, API response none

**Comment by NelsonDane** [comment]
Same here

**Pull request merged**